consumer/fanoutconsumer/cloningconsumer.go

```diff
@@ -36,6 +36,10 @@ type metricsCloningConsumer []consumer.Metrics
 
 var _ consumer.Metrics = (*metricsCloningConsumer)(nil)
 
+func (mfc metricsCloningConsumer) Capabilities() consumer.Capabilities {
+	return consumer.Capabilities{MutatesData: true}
```

Comment:
The cloning consumer does not mutate the data, right? Should this be `true`? Or alternatively you can say that this consumer does not mutate the data but the net one may mutate it.

Reply:
Cloning mutates one of the copy, it does N-1 copies and passes the Nth one to the last consumer which currently we don't know if mutates or not. This can be improved once we have the Consumer capability, but I will do it next